screens/ProfileScreen.tsx

```diff
@@ -27,7 +38,7 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'flex-start',
     paddingVertical: 18,
-    paddingHorizontal: 24
+    paddingHorizontal: 18
```

Comment:
Can you change this to just be padding since you have paddingVertical and paddingHorizontal?